docs/projects/honest_calculator.md

```
@@ -0,0 +1,108 @@
+
+    This project is my implementation of the **[Honest Calculator](https://hyperskill.org/projects/208)** project from the Hyperskill platform.
+
+    The premise is fun: I had to build a calculator for a fictional "International Union Against Idleness" competition. The goal wasn't just to calculate, but to motivate users. The calculator helps with complex math, but for simple operations—like adding small numbers or multiplying by one—it gets a bit "honest" and encourages the user to do it themselves. This quirky requirement makes it a great challenge for handling state and conditional logic.
+
+    > For me, this project was the perfect opportunity to move beyond a simple script. The need to manage memory, a series of "laziness" checks, and a multi-step confirmation flow pushed me to refactor the initial procedural code into a much cleaner, class-based design. It became a fantastic case study in making code not just work, but be testable and readable.
+
+## Usage
+
+To run the calculator, use the following command in your terminal after installation:
+
+```shell
```

markdownlint-cli2 (0.17.2)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)